### PR TITLE
Functional tests - View customer

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add test for view customer
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/04_customers/01_customers/08_viewCustomer" URL_FO=yourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18754)

### CODE REVIEW

File: tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/08_viewCustomer.js [hunk 1]
```diff
@@ -0,0 +1,529 @@
...
+      await expect(cardHeaderText).to.contains(createCustomerData.firstName);
+      await expect(cardHeaderText).to.contains(createCustomerData.lastName);
+      await expect(cardHeaderText).to.contains(createCustomerData.email);
+    });
+
+    it('should check customer personal information', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkCreatedCustomerInfo1', baseContext);
+      const cardHeaderText = await this.pageObjects.viewCustomerPage.getTextFromElement('Personal information');
+      await expect(cardHeaderText).to.contains(createCustomerData.socialTitle);
+      await expect(cardHeaderText).to.contains(`birth date: ${customerBirthDate}`);
```
Comment (Progi1984) [inline tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/08_viewCustomer.js:158]: Not began by a uppercase ?

File: tests/puppeteer/pages/BO/customers/view.js [hunk 2]
```diff
@@ -6,22 +6,173 @@ module.exports = class ViewCustomer extends BOBasePage {
...
+        selector = this.groupsDiv;
+        break;
+      default:
+        throw new Error(`${element} was not found`);
+    }
+    return this.getTextContent(`${selector} .card-body`);
+  }
+
+  /**
+   * Se private note
```
Comment (SimonGrn) [inline tests/puppeteer/pages/BO/customers/view.js]: ?? :laughing:

File: tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/08_viewCustomer.js [hunk 3]
```diff
@@ -0,0 +1,529 @@
...
+      await expect(cardHeaderText).to.contains('Active');
+    });
+
+    const tests = [
+      {args: {blockName: 'Orders', number: 1}},
+      {args: {blockName: 'Carts', number: 1}},
+      {args: {blockName: 'Viewed products', number: 1}},
+      {args: {blockName: 'Messages', number: 1}},
+      {args: {blockName: 'Vouchers', number: 0}},
+      // {args: {blockName: 'Last emails', number: 2}},
```
Comment (SimonGrn) [inline tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/08_viewCustomer.js]: You can remove this line

Comment (nesrineabdmouleh) [reply tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/08_viewCustomer.js]: It was a bug :) now it fixed I will delete the //

File: tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/08_viewCustomer.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/08_viewCustomer.js:158]: No it not began by an uppercase in the form

Comment (Progi1984) [reply tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/08_viewCustomer.js:158]: :ok_hand:

File: tests/puppeteer/pages/FO/checkout/index.js [hunk 4]
```diff
@@ -35,6 +33,11 @@ module.exports = class Checkout extends FOBasePage {
     this.addressStepCountrySelect = `${this.addressStepSection} select[name='id_country']`;
     this.addressStepPhoneInput = `${this.addressStepSection} input[name='phone']`;
     this.addressStepContinueButton = `${this.addressStepSection} button[name='confirm-addresses']`;
+    // Shipping method step
+    this.deleveryStepSection = '#checkout-delivery-step';
```
Comment (Progi1984) [inline tests/puppeteer/pages/FO/checkout/index.js]:
```suggestion
    this.deliveryStepSection = '#checkout-delivery-step';
```

Comment (nesrineabdmouleh) [reply tests/puppeteer/pages/FO/checkout/index.js]: Done thx :)

**Review by Progi1984 — APPROVED** [review]

**Review by SimonGrn — APPROVED** [review]

**Pull request merged**